zanata-war/src/main/webapp/WEB-INF/layout/project/people-tab.xhtml

```diff
@@ -0,0 +1,202 @@
+        <a4j:status name="peopleTab-peopleLoader">
+          <f:facet name="start">
+            <z:loader layout="inline" type="loader--small"/>
+          </f:facet>
+        </a4j:status>
+      </h2>
+    </div>
+
+    <!-- Search row -->
+    <div class="panel__sub-header js-reveal">
+      <z:list-filter status="peopleTab-peopleLoader"
+        listId="people_form" iconClass="i--users"
+        render="people_form, peopleSearchBottom-pager, peopleSearchBottom-page-info"
+        id="peopleSearch"
+        placeholder="#{msgs['jsf.people.SearchPlaceholder']}"
+        actionBean="#{projectHomeAction.peopleFilterComparator}"/>
+    </div>
+
+    <!-- List of people -->
+    <h:form id="people_form" styleClass="l--push-bottom-1">
+
+      <!-- No people message -->
+      <s:div styleClass="l--pad-all-half"
+        rendered="#{empty projectHomeAction.allMembers}">
+        <p class="txt--meta">#{msgs['jsf.NoPeople']}</p>
+        <s:fragment rendered="#{canManageMembers}">
+          <p>
+            <z:ajax-command-button
+                styleClass="button button--primary panel__header__button"
+                action="#{projectPermissionDialog.setData(projectHome.instance, null)}"
+                render="modalManagePermissions-content"
+                oncomplete="zanata.modal.show(jQuery('#modalManagePermissions'))">
+              <i class="i i--plus"></i> #{msgs['jsf.people.AddSomeone']}
+              <f:facet name="loading">
+                <i class="i i--plus"></i> #{msgs['jsf.people.AddSomeone']}
+              </f:facet>
+            </z:ajax-command-button>
+          </p>
+        </s:fragment>
+      </s:div>
+
+      <!-- Alphabetical people list -->
+      <s:fragment rendered="#{not projectHomeAction.peopleFilterComparator.isShowMembersInGroup()}">
+        <s:div styleClass="l--pad-all-half"
+          rendered="#{not empty projectHomeAction.allMembers and empty projectHomeAction.peopleFilterComparator.currentPage}">
+          <p class="txt--meta">#{msgs['jsf.search.NoResult']}</p>
+        </s:div>
+
+        <s:fragment
+          rendered="#{not projectHomeAction.peopleFilterComparator.currentPage.isEmpty()}">
+          <ul class="list--slat l--push-all-half">
+            <ui:repeat value="#{projectHomeAction.peopleFilterComparator.currentPage}" var="person">
+              <z:person-permission-entry person="#{person}"
+                project="#{projectHomeAction.project}"
+                roleNames="#{projectHomeAction.allRoleDisplayNames(person)}"
+                canManageMembers="#{canManageMembers}"
+                managePermissionsRender="modalManagePermissions-content"
+                managePermissionsModal="modalManagePermissions"/>
+            </ui:repeat>
+          </ul>
+        </s:fragment>
+      </s:fragment>
+
+      <!-- People list grouped by role -->
+      <s:fragment rendered="#{projectHomeAction.peopleFilterComparator.isShowMembersInGroup()}">
```

Comment:
Sorry David this is not your fault but we are moving away from Seam taglib now. Please see https://github.com/zanata/zanata-server/pull/928 for replacement
- [ ] replace seam tag